movement/watch_faces/clock/beats_face.c

```diff
@@ -1,3 +1,27 @@
+/*
+ * MIT License
```

Comment:
Adding copyright info to code you didn't write seems dubious to me. @mrbrown8, can you confirm this?

Reply:
Thank you for asking, but I did NOT write this code. I cannot take credit/blame for this.

Reply:
@WesleyAC you are right.  In hindsight, that was awfully presumptuous of me.  Sorry!

For the Beats face, @tahnok is the primary author and has confirmed MIT licensing info in a comment below.  I don't know what I was thinking when I put @mrbrown8 , but I will fix that now.